sharepoint/sharepoint-ps/sharepoint-online/Get-SPOUnifiedGroupMoveState.md

```diff
@@ -103,6 +103,9 @@ The move status for the Office 365 Group. Possible values are:
 * InProgress (n/4): The move is in progress in one of the following states: Validation (1/4), Backup (2/4), Restore (3/4), Cleanup (4/4).
 * Success: The move has completed successfully.
 * Failed: The move failed.
+* Stopped: The move was canceled by an admin while it was still queued.
+* NotSupported: The move could not be processed because the PDL was invalid.
```

Comment:
>PDL [](http://example.com/codeflow?start=60&length=3)

only use an acronym after it's been spelt out somewhere in this doc. otherwise spell it out here.

Reply:
Updated